cheatsheets_draft/Authorization_Cheat_Sheet.md

```diff
@@ -0,0 +1,47 @@
+# Authorization Cheat Sheet
+
+## Introduction
+
+Focus on definition and distinguishing it from authentication. Perhaps explore types of privilege escalation (horizontal and vertical). Not sure if we should briefly explore the different access control models (RBAC, MAC, DAC, etc) or just leave it out?
+
+## Prevalence and Impact
+
+Try and answer the question of why the reader (developer) should care about the topic.
+
+## Attack Scenarios
+
+Would include vulnerable code examples here.
```

Comment:
Not sure what kind of vulnerable code you'd include here?

Reply:
I wouldn't dig too much on this right now? It could maybe discuss how validation is being checked, or bad practices of roles. I'd say that this section is going contain threats (like threat models we are setting in other CSs), so maybe rewording the title into something more towards threats, and containing examples if need be?

Reply:
For right now I just removed this, but agreed it might be a good idea to replace with a discussion of threat models. Sorry, "vulnerable code" was not very clear in hindsight. Just meant examples of code which would be vulnerable to access control related attacks in general. Probably would be better to locate in any such examples under the appropriate "Recommendation" subsection.